docs/command-reference/commands-details/workload_create_update_apply.md

```diff
@@ -0,0 +1,1154 @@
+# tanzu apps workload apply
+
+tanzu apps workload apply is a command used to create/update workloads that will be deployed in a cluster through a supply chain.
+
+## Default view
+
+In the output of workload apply command, the specification for the workload will be shown as if they were in a yaml file.
+
+<details><summary>Example</summary>
+
+```bash
+tanzu apps workload apply pet-clinic --git-repo https://github.com/sample-accelerators/spring-petclinic --git-tag tap-1.1 --type web
+
+Create workload:
+    1 + |---
+    2 + |apiVersion: carto.run/v1alpha1
+    3 + |kind: Workload
+    4 + |metadata:
+    5 + |  labels:
+    6 + |    apps.tanzu.vmware.com/workload-type: web
+    7 + |  name: pet-clinic
+    8 + |  namespace: default
+    9 + |spec:
+   10 + |  source:
+   11 + |    git:
+   12 + |      ref:
+   13 + |        tag: tap-1.1
+   14 + |      url: https://github.com/sample-accelerators/spring-petclinic
+
+? Do you want to create this workload? Yes
+Created workload "pet-clinic"
+
+To see logs:   "tanzu apps workload tail pet-clinic"
+To get status: "tanzu apps workload get pet-clinic"
+
+```
+</details>
+
+In the first part, the workload definition is displayed. After that, there is a survey asking user if the workload should be created or updated and finally, if workload is actually to be created or updated, a couple of hints to commands that can be used to do a follow up. Each flag used in this example will be explained in detail in the following section.
```

Comment:
In the first section, the definition of workload is displayed. Its followed by a prompt asking whether the workload should be created or updated. In the last section, if workload is actually to be created or updated, a couple of hints/suggestions are displayed about the next set of commands that can be used for a follow up. Each flag used in this example will be explained in detail in the following section.